src/__test__/handle-data.spec.ts

```diff
@@ -214,6 +214,10 @@ describe('handle-data', () => {
       layout.totals.label = '';
       expect(getTotalInfo(layout, 0, 0, 2)).toBe('');
     });
+
+    it('should return first dimension total value as Totals when colIdx is not 0', () => {
```

Comment:
Can we get Total values on dimensions? I was thinking totals row is only for when we have measures

Reply:
That would be correct I think. So it should also be a condition that measures.length > 0